.github/workflows/cut-release-branch.yml

```diff
@@ -294,7 +299,8 @@ jobs:
         run: |
           source env/bin/activate
           python -m pip install -r dev-requirements.txt
-          env/bin/bumpversion --allow-dirty --new-version ${{ inputs.version_to_bump_main }} major
+          env/bin/bumpversion --allow-dirty \ 
+           --new-version : ${{ fromJSON(needs.create_temp_branch.outputs.next-version) }} major
```

Comment:
Does the colon do something here?  Also why is next-version not just a string?  Why is it json?

Reply:
nope, just forgot to remove after I copied it from somewhere that needed it